index.js

```diff
@@ -1,5 +1,99 @@
-console.log("hola");
+"use strict";
+const listOfCards = document.querySelector("#lista");
+const shuffle = document.querySelector(".shuffle");
+const topCard = document.querySelector("#top-card");
 
-const mainContainer = document.querySelector("#app");
+//GENERANDO ARRAY CON CARTAS
 
-mainContainer.innerText = "Hello World";
+const cardGeneration = function () {
+  const cardsArrGen = [];
+  const clubs = ["clubs", "diamonds", "hearts", "spades"];
+  const cards = ["As", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"];
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
Let's avoid using magic numbers/values there is no way for other people to know what this number `4` represent, in this case I think we can replace it with the length of the `clubs` array
```suggestion
  for (let i = 0; i < clubs.length; i++) {
```